.circleci/config.yml

```diff
@@ -23,49 +23,22 @@ commands:
               echo "No RBE credential found. Bazel will be executed locally without RBE support."
             fi
 
-  bazel_execute:
+  bazel:
     parameters:
-      parameters:
+      command:
           type: string
     steps:
       - run:
           name: Bazel - Execute
           command: |
             if [[ -f ~/.config/gcloud/application_default_credentials.json ]]; then
               echo "Bazel will be executed with RBE support. This means the build is remotely executed and the cache will be re-used by subsequent CI jobs."
-              RBE_FLAGS="--project_id=grakn-dev \
-                --remote_instance_name=projects/grakn-dev/instances/default_instance \
-                --remote_cache=remotebuildexecution.googleapis.com \
-                --remote_executor=remotebuildexecution.googleapis.com \
-                --bes_backend="buildeventservice.googleapis.com" \
-                --bes_results_url="https://source.cloud.google.com/results/invocations/" \
-                --host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:rbe_ubuntu1604 \
-                --platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:rbe_ubuntu1604 \
-                --extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:rbe_ubuntu1604 \
-                --host_javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:jdk8 \
-                --javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:jdk8 \
-                --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8 \
-                --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8 \
-                --extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/1.1/bazel_0.20.0/cpp:cc-toolchain-clang-x86_64-default \
-                --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/1.1/bazel_0.20.0/default:toolchain \
-                --jobs=50 \
-                --remote_timeout=3600 \
-                --bes_timeout=60s \
-                --tls_enabled=true \
-                --auth_enabled=true \
-                --spawn_strategy=remote \
-                --strategy=Javac=remote \
-                --strategy=Closure=remote \
-                --genrule_strategy=remote \
-                --define=EXECUTOR=remote \
-                --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1 \
-                --experimental_strict_action_env=true
-                "
+              RBE_FLAGS="--config=rbe"
```

Comment:
We should inline this `RBE_FLAGS` variable since it's so short now. I.e. just do 
```
if credentials exist
    CMD="<< parameters.command >> --config=rbe"
else
    CMD="<< parameters.command >>"
fi
```

Reply:
done: https://github.com/graknlabs/grakn/pull/4780/files#diff-1d37e48f9ceff6d8030570cd36286a61